airbyte-webapp/src/views/Connection/ConnectionForm/ConnectionForm.tsx

```diff
@@ -265,7 +204,11 @@ export const ConnectionForm: React.FC<ConnectionFormProps> = ({
                       error={!!meta.error && meta.touched}
                       options={frequencies}
                       onChange={(item) => {
-                        onDropDownSelect?.(item);
+                        // This was only passed in for the create view
+                        // Do we want it for all?
+                        if (mode === "create") {
+                          onFrequencySelect(item);
+                        }
```

Comment:
Adding visibility to this question

Reply:
Tagging @nataliekwong on that one!

Currently we only trigger the Frequency Select analytics call when creating a connection. The alternative here is we could send analytics whenever a user selects a frequency (ie: creating or editing).

If we do change it, I imagine we will also want to track the mode (create or edit) in the call name.

Reply:
Yep, agreed with @tealjulia 's comment on splitting the call into create and edit separately.

Reply:
Would you be okay with me just including the `mode` in the call? As part of the Event?

```
        analyticsService.track(Namespace.CONNECTION, Action.FREQUENCY, {
          actionDescription: "Frequency selected",
          frequency: item.label,
          connector_source_definition: connection.source.sourceName,
          connector_source_definition_id: connection.source.sourceDefinitionId,
          connector_destination_definition: connection.destination.destinationName,
          connector_destination_definition_id: connection.destination.destinationDefinitionId,
          available_streams: connection.syncCatalog.streams.length,
          enabled_streams: enabledStreams,
          mode: mode,
        });
        ```

Reply:
Yes, it's fine to use that. Could we rename it from "mode" to "type"? Are the values for it going to be "new" and "existing"? Or "create" and "update"?

And just to double check, the name the call is going to remain as "airbyte_ui_cloud.airbyte_ui_frequency_set"? (Not frequency selected)

Reply:
The name should remain the same. I'll pass `type` instead of mode. The value will be one of these: `"create" | "edit" | "readonly";`
I could transcribe create to "new" and edit to "existing". Technically readonly should never be sent.